src/main/java/it/gov/pagopa/cgn/portal/email/EmailNotificationFacade.java

```diff
@@ -137,11 +157,15 @@ public void notifyMerchantDiscountExpiring(String referentEmail, String discount
         }
     }
 
-    public static String createTrackingKeyForExiprationNotification(DiscountEntity discount, BucketCodeExpiringThresholdEnum threshold) {
+    public static String createTrackingKeyForExiprationNotification(DiscountEntity discount,
```

Comment:
typo

Reply:
Fixed in https://github.com/pagopa/cgn-onboarding-portal-backend/pull/134/commits/6f467bbf40782949d4abd00f6d83684ee272e870